week-5-practise/js/main.js

```diff
@@ -0,0 +1,103 @@
+/*### part 1
+
+whhen clicking **blue** it should change:
+        - **Jumbotron** background color to `#588fbd`
+        - **Donate a bike** button background color to `#ffa500`
+        - **Volunteer** button background color to `black` and text color to `white`*/
+
+var blueTheme = document.querySelector("#blueBtn");
+var myContainer = document.querySelector(".jumbotron");
+var myButton = document.querySelectorAll(".btn-lrg");
```

Comment:
These are all right! Just a minor detail: variables assigned an array should have a plural name, e.g. `myButtons`.